conf/evolutions/39.js

```diff
@@ -0,0 +1,34 @@
+// Turn annotation flags into an enum
+
+// --- !Ups
+db.annotations.find().forEach(function(elem) {
+  let newState = ""
```

Comment:
Can you please rename the old `state` entry first, so that we have a chance to roll back?